src/browser-polyfill.js

```diff
@@ -297,16 +324,18 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
         },
       };
 
-      // Per contract of the Proxy API, the "get" proxy handler must return the
-      // original value of the target if that value is declared read-only and
-      // non-configurable. For this reason, we create an object with the
-      // prototype set to `target` instead of using `target` directly.
-      // Otherwise we cannot return a custom object for APIs that
-      // are declared read-only and non-configurable, such as `chrome.devtools`.
-      //
-      // The proxy handlers themselves will still use the original `target`
-      // instead of the `proxyTarget`, so that the methods and properties are
-      // dereferenced via the original targets.
+      /**
+       * Per contract of the Proxy API, the "get" proxy handler must return the
+       * original value of the target if that value is declared read-only and
+       * non-configurable. For this reason, we create an object with the
+       * prototype set to `target` instead of using `target` directly.
+       * Otherwise we cannot return a custom object for APIs that
+       * are declared read-only and non-configurable, such as `chrome.devtools`.
+       *
+       * The proxy handlers themselves will still use the original `target`
+       * instead of the `proxyTarget`, so that the methods and properties are
+       * dereferenced via the original targets.
+       */
```

Comment:
Converted comment to&nbsp;JSDoc.

Reply:
Why? This comment is just explaining the rationale for internal details and not relevant to API consumers.